docs/migration/4_0.md

```diff
@@ -783,6 +783,61 @@ The following properties where removed in 4.0:
 - Styles for following selectors `cx-close-account-modal`, `cx-close-account`, `cx-forgot-password`, `cx-register`, `cx-update-password-form`, `cx-user-form` were moved from `@spartacus/styles` to `@spartacus/user/profile/styles`.
 - Styles for following selectors `cx-login`, `cx-login-form` were moved from `@spartacus/styles` to `@spartacus/user/account/styles`.
 
+### (a11y) Screen Reader Improvements: Headings
+- Most of these changes affect only headings on screen reader rotor feature. So changes are visible for sighted users. 
+#### StoreFinderListItemComponent
+- `div[class.cx-store-name]` element has been changed to `h2[class.cx-store-name]`.
+#### StoreFinderStoresCountComponent
+- `div[class.cx-title` element has been changed to `h2[class.cx-title]`
+
+#### div[class.cx-total]` element has been chanted to `h4[class.cx-total]`
+
+#### CartItemComponent
+`{{item.product.name}}` element has been changed to `<h2>{{item.product.name}}</h2>`
+
+#### OrderSummaryComponent
+- `<h4>{{ 'orderCost.orderSummary' | cxTranslate }}</h4>` element has changed to `<h3>{{ 'orderCost.orderSummary' | cxTranslate }}</h3>`
+
+#### DeliveryModeComponent
+- `h3[class.cx-checkout-title]` element has changed to `h2[class.cx-checkout-title]`
+
+#### PaymentMethodComponent
+- `h3[class.cx-checkout-title]` element has changed to `h2[class.cx-checkout-title]`
+
+#### ReviewSubmitComponent
+- `h3[class.cx-review-title]` element changed to `h2[class.cx-review-title]`
+- `div[class.cx-review-cart-total]` element changed to `h4[class.cx-review-cart-total]`
+
+#### ShippingAddressComponent
+- `h3[class.cx-checkout-title]` element changed to `h2[class.cx-checkout-title]`
+
+#### BreadcrumbComponent
+- `isNotNullable`, `filter` import was removed 
+- `PageTitleComponent` import was added
+- `BreadcrumbComponent` extends `PageTitleComponent`
+- `setTitle()` function was moved to `PageTitleComponent`
+
+#### NavigationUiComponent
+- `<h5><cx-icon ...></h5>` element was changed to `<span><cx-icon ...></span>`
+- `h5[attr.aria-label]="node.title"` element was was changed to `span[attr.aria-label]="node.title"`
+
+#### PageTitleComponent 
+- New component that sets page title if there is not one set by default 
+
+#### ProductCarouselComponent
+- `<h4>{{item.name}}</h4>` element changed to `<h3>{{item.name}}</h3>`
+
+#### WishListItemComponent 
+- `<a>{{ cartEntry.product.name }}</a>` element changed to `<a><h2>{{ cartEntry.product.name }}</h2></a>`
+
+#### CardComponent
+- `h4[class.cx-card-title]` element changed to `h3[class.cx-card-title]`
+
+#### CarouselComponent
+- `<h3 *ngIf="title">{{ title }}</h3>` element changed to `<h2 *ngIf="title">{{ title }}</h2>`
```

Comment:
I would like to see it below other `component` change and above `###Translations (i18n) changes`